src/cljam/util/sam_util.clj

```diff
@@ -330,8 +334,8 @@
 (defn ref-name
   "Returns a reference name from the reference ID. Returns nil if id is not
   mapped."
-  [refs id]
-  (if (<= 0 id (dec (count refs)))
+  [refs ^long id]
+  (when (<= 0 id (dec (count refs)))
```

Comment:
I personally think existence of side effects is more important than whether the function return `nil`. The third argument of `if` is not a serious matter. Well, OK in this case because style guide about `if/when` hasn't been established yet.